### PR TITLE
opengist: 1.8.3 -> 1.8.4

### PR DESCRIPTION
**Pull request by phanirithvij**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [x] x86_64-darwin
  - [x] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [x] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by phanirithvij** [comment]
@ofborg build opengist

File: pkgs/by-name/op/opengist/package.nix [hunk 1]
```diff
@@ -11,25 +11,17 @@
 let
   # finalAttrs when 🥺 (buildGoModule does not support them)
   # https://github.com/NixOS/nixpkgs/issues/273815
-  version = "1.8.3";
+  version = "1.8.4";
   src = fetchFromGitHub {
     owner = "thomiceli";
     repo = "opengist";
     rev = "v${version}";
```
Comment (GaetanLepage) [inline pkgs/by-name/op/opengist/package.nix:18]:
```suggestion
    tag = "v${version}";
```
If you don't mind, please switch to the new `tag` syntax.
Also, change the `meta.changelog` to `"https://github.com/thomiceli/opengist/releases/tag/${src.tag}"`.

Comment (phanirithvij) [reply pkgs/by-name/op/opengist/package.nix:18]: changelog.md is fine, no? it is up to date.

Comment (GaetanLepage) [reply pkgs/by-name/op/opengist/package.nix]: Sorry, I meant `"https://github.com/thomiceli/opengist/blob/${src.tag}/CHANGELOG.md"`

Comment (phanirithvij) [reply pkgs/by-name/op/opengist/package.nix]: done

**Comment by phanirithvij** [comment]
@ofborg build opengist

**Comment by GaetanLepage** [comment]
```
buildPhase completed in 1 minutes 32 seconds
Running phase: checkPhase
fatal: not a git repository (or any of the parent directories): .git
?       github.com/thomiceli/opengist   [no test files]
?       github.com/thomiceli/opengist/internal/actions  [no test files]
?       github.com/thomiceli/opengist/internal/auth     [no test files]
?       github.com/thomiceli/opengist/internal/auth/totp        [no test files]
?       github.com/thomiceli/opengist/internal/auth/webauthn    [no test files]
?       github.com/thomiceli/opengist/internal/cli      [no test files]
?       github.com/thomiceli/opengist/internal/config   [no test files]
?       github.com/thomiceli/opengist/internal/db       [no test files]
ok      github.com/thomiceli/opengist/internal/git      3.363s
panic: open /etc/protocols: operation not permitted

goroutine 1 [running]:
modernc.org/libc/honnef.co/go/netdb.init.0()
        modernc.org/libc@v1.61.2/honnef.co/go/netdb/netdb.go:700 +0x640
FAIL    github.com/thomiceli/opengist/internal/hooks    0.323s
?       github.com/thomiceli/opengist/internal/i18n     [no test files]
?       github.com/thomiceli/opengist/internal/i18n/locales     [no test files]
?       github.com/thomiceli/opengist/internal/index    [no test files]
?       github.com/thomiceli/opengist/internal/memdb    [no test files]
?       github.com/thomiceli/opengist/internal/render   [no test files]
?       github.com/thomiceli/opengist/internal/ssh      [no test files]
?       github.com/thomiceli/opengist/internal/utils    [no test files]
?       github.com/thomiceli/opengist/internal/web      [no test files]
panic: open /etc/protocols: operation not permitted

goroutine 1 [running]:
modernc.org/libc/honnef.co/go/netdb.init.0()
        modernc.org/libc@v1.61.2/honnef.co/go/netdb/netdb.go:700 +0x640
FAIL    github.com/thomiceli/opengist/internal/web/test 0.363s
?       github.com/thomiceli/opengist/public    [no test files]
?       github.com/thomiceli/opengist/templates [no test files]
FAIL
make: *** [Makefile:76: test] Error 1
```

**Comment by phanirithvij** [comment]
I don't have access to macos, so I can't verify it myself.

**Comment by GaetanLepage** [comment]
> I don't have access to macos, so I can't verify it myself.

It fails, just skip those tests on `darwin` I would say.

**Comment by GaetanLepage** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 365507`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>opengist</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>opengist</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>opengist</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>opengist</li>
  </ul>
</details>

**Comment by phanirithvij** [comment]
@GaetanLepage any clue why it builds fine on ofborg for x86_64-darwin? https://logs.ofborg.org/?key=nixos/nixpkgs.365507&attempt_id=455b2625-8c63-45b2-a572-883e065c944c

**Comment by GaetanLepage** [comment]
> @GaetanLepage any clue why it builds fine on ofborg for x86_64-darwin? https://logs.ofborg.org/?key=nixos/nixpkgs.365507&attempt_id=455b2625-8c63-45b2-a572-883e065c944c

Probably a sandbox issue.

**Comment by GaetanLepage** [comment]
What is this `/etc/protocols` ? Why does `opengist` needs to access it ? Can we patch this in the source code instead ?

**Comment by GaetanLepage** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 365507`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>opengist</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>opengist</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>opengist</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>opengist</li>
  </ul>
</details>

File: pkgs/by-name/op/opengist/package.nix [hunk 2]
```diff
@@ -7,29 +7,22 @@
   moreutils,
   jq,
   git,
+  stdenv,
```
Comment (GaetanLepage) [inline pkgs/by-name/op/opengist/package.nix:10]: Please, put `stdenv` at the top, right after `lib`.

**Comment by GaetanLepage** [comment]
> they disable checks for darwin. so I did the same, disabled check for darwin.

Thanks for looking into it !

File: pkgs/by-name/op/opengist/package.nix (hunk 2 above)
Comment (phanirithvij) [reply pkgs/by-name/op/opengist/package.nix]: done

**Review by GaetanLepage — APPROVED** [review]

**Comment by GaetanLepage** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 365507`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>opengist</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>opengist</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>opengist</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>opengist</li>
  </ul>
</details>

**Comment by phanirithvij** [comment]
Thank you, I didn't realize that ofborg had sandbox relaxed.

**Pull request merged**